stories/Uploader.stories.tsx

```diff
@@ -22,7 +25,10 @@ const Uploader = args => {
             multiple: true,
             limit: 5,
             onFileClick: file => void 0,
-            onFilesChange: files => void 0,
+            onFilesChange: files => {
+                console.log('files', files)
```

Comment:
I think the console log is not necessary.

Reply:
it's to test the callback in storybook so we should keep it